alpine-keys.yaml

```diff
@@ -5,9 +5,9 @@ package:
   description: Public keys for Alpine Linux packages
   copyright:
     - license: MIT
-  dependencies:
-    runtime:
-      - ca-certificates-bundle
+  options:
+    # There are no executables in this package.
```

Comment:
I don't think you need to add this section. IIUC, melange manages these fields in the built APK and APK index automatically.

This will instruct melange not to search for commands, the end result should be the same but the default behavior in melange would be safer.